pkg/problemmetrics/problem_metrics_test.go

```diff
@@ -43,7 +43,7 @@ func TestNewProblem(t *testing.T) {
 			counts:  []int64{1},
 			expectedMetrics: []metrics.Int64MetricRepresentation{
 				{
-					Name:   "problem_counter",
+					Name:   "problem_count_total",
```

Comment:
count and total may seem quite similar? What about problem_occurrence_total?

Reply:
I have no preference about this :) `problem_total` or `problem_count_total` or `problem_occurrence_total` all looks fine to me :)

@HaveFun83 proposed `problem_count_total` [here](https://github.com/kubernetes/node-problem-detector/issues/326#issuecomment-521135338).

Reply:
That's OK. Let's choose problem_count_total since problem_total is kind of ambiguous.